src/api/Services/FolderOperations.cs

```diff
@@ -16,11 +18,10 @@ internal class FolderOperations
 		private readonly ILogger log;
 		private readonly DataLakeFileSystemClient dlfsClient;
 		private readonly decimal costPerTB;
-		private readonly TokenCredential tokenCredential;
+
 		public FolderOperations(ILogger log, TokenCredential tokenCredential, Uri storageUri, string fileSystem)

```

Comment:
Why is `TokenCredential` still being injected in the ctor?

Reply:
By injecting it in the constructor, every call within the instance object will be of the correct context. If you wish to create a second instance in another context, you can.

Reply:
Sure, that makes sense, but the instance-level object isn't there anymore. So the ctor gets it, but it just goes to a black hole?